juniper_official/routing/collect-isis-lsdb-lsp.rule

```diff
@@ -0,0 +1,98 @@
+/*
+ * Collects LSDB data for lsp-install-time and lsp sequence number
+ */
+ healthbot {
+    topic routing.isis {
+        rule collect-isis-lsdb-lsp {
+            keys [ instance-name protocol-name level-numb lsp-id ];
+            synopsis "Collects ISIS lsp-intall-time,sequence-number";
+            description "Collects LSDB path periodically till lsp node as key";
+            sensor isis-oc {
+            	synopsis "ISIS open-config sensor definition";
+                description "Open-config sensor to collect telemetry data from network device";
+                open-config {
+                    sensor-name /network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/;
+                    frequency 60s;
+                }
+            }			
+            field instance-name {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/@name =~ /{{inst-name}}/";
+                    path "/network-instances/network-instance/@name";
+                }			
+                type string;
+                description "instance-name from the lsdb path";
+            }
+            field protocol-name {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/protocols/protocol/@identifier=~/{{proto-name}}/";
+                    path "/network-instances/network-instance/protocols/protocol/@identifier";
+                }			
+                type string;
+                description "protocol-name from the lsdb path";
+            }
+            field level-numb {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/protocols/protocol/isis/levels/level/@level-number=~/{{level-num}}/";
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/@level-number";
+                }
+                type string;
+                description "level-number from the lsdb path";
+            }
+            field lsp-id {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/@lsp-id";
+                }
+                type string;
+                description "lsp-id from the lsdb path";
+            }
+            field lsp-install-time {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/state/lsp-install-time";
+                }
+                type string;
```

Comment:
What is the format of the value that comes here?

Reply:
Modified to unsigned-integer